xarray/tests/test_sparse.py

```diff
@@ -707,7 +707,7 @@ def test_dataarray_repr(self):
             <xarray.DataArray (x: 4)>
             <COO: shape=(4,), dtype=float64, nnz=4, fill_value=0.0>
             Coordinates:
-                y        (x) int64 <COO: shape=(4,), nnz=3, fill_value=0>
+                y        (x) int64 <COO: nnz=3, fill_value=0>
```

Comment:
isn't it sparse.COO now?

Reply:
I was sticking with a simplified version of the original repr here, only putting `sparse.COO` inside `meta`.